### PR TITLE
Fix the code for some parts

### PR DESCRIPTION
**Pull request by MichaelKim20**
Change to `let` instead of `var`
Add missing semicolons
Change deprecated `equal` to `strictEqual`
Changed to call a callback at the end of the test

### CODE REVIEW

File: tests/Storage.test.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import * as assert from 'assert';
 import { LedgerStorage } from '../src/modules/storage/LedgerStorage';
 import { Enrollment } from '../src/modules/data';
 
-var sample_data =
+let sample_data =
```
Comment (TrustHenry) [inline tests/Storage.test.ts]:
```suggestion
const sample_data =
```
If you change it, const seems suitable.

Comment (MichaelKim20) [reply tests/Storage.test.ts]: Okay

**Review by TrustHenry — APPROVED** [review]
Nice fix.
LGTM

File: tests/Storage.test.ts (hunk 1 above)
Comment (MichaelKim20) [reply tests/Storage.test.ts]: I fixed it, thank you for review.

**Review by Geod24 — APPROVED** [review]

File: tests/Storage.test.ts [hunk 2]
```diff
@@ -521,18 +522,18 @@ function runValidatorsAPITest (ledger_storage: LedgerStorage, onDone: () => void
...
 describe('LedgerStorage', () =>
 {
-    it('Test ledger storage and inquiry function.', () =>
+    it('Test ledger storage and inquiry function.', (doneIt: () => void) =>
     {
-        runLedgerStorageTest();
+        runLedgerStorageTest(doneIt);
     });
-    it ('Test validation of JSON data', () =>
+    it ('Test validation of JSON data', (doneIt: () => void) =>
```
Comment (TrustHenry) [inline tests/Storage.test.ts:529]:
```suggestion
    it('Test validation of JSON data', (doneIt: () => void) =>
```

**Pull request merged**